README.md

```diff
@@ -201,6 +201,51 @@ A custom tool definition for `helm` could look like the following example:
     Use `helm --help` or `helm <subcommand> --help` to see full syntax, available flags, and examples for each command.
 ```
 
+## MCP Client Mode
+
+`kubectl-ai` can act as an MCP (Model Context Protocol) client, connecting to external MCP servers to access additional tools and capabilities beyond the built-in ones.
```

Comment:
I think we can simply say:

`kubectl-ai` can connect to external [MCP](http://mcp...) Servers to access additional tools in addition to built-in tools.

I think at some point, we will move them under one section `Using other tools` section